src/themes/default/elements/placeholder.variables

```diff
@@ -34,16 +34,8 @@
 /* Glow Gradient */
 @placeholderLoadingAnimationDuration: 2s;
 @placeholderLoadingGradientWidth: 1200px;
-@placeholderLoadingGradient: linear-gradient(to right,
-rgba(0, 0, 0, 0.08) 0,
-rgba(0, 0, 0, 0.15) 15%,
-rgba(0, 0, 0, 0.08) 30%
-);
-@placeholderInvertedLoadingGradient: linear-gradient(to right,
-rgba(255, 255, 255, 0.08) 0,
-rgba(255, 255, 255, 0.14) 15%,
-rgba(255, 255, 255, 0.08) 30%
-);
+@placeholderLoadingGradient: linear-gradient(to right, rgba(0, 0, 0, 0.08) 0, rgba(0, 0, 0, 0.15) 15%, rgba(0, 0, 0, 0.08) 30%);
```

Comment:
Why is this now concatenated into one line while in other cases (box-shadow, every list in variation.variables, font-family, etc. )using comma separated lists, they are kept? Sometimes it helps for better readability, sometime not :(

Reply:
https://github.com/fomantic/Fomantic-UI/pull/2611/commits/470cd4e7b1dbce900d5e006aa8f3c5d7bf850009